### ISSUE TITLE
Multiple verilator instances of SystemC models in one single simulation

### ISSUE THREAD

**Issue by robertbu** [verilator/verilator#5702]
Hi,
I have multiple RTL sources from different vendors. It's not possible to integrate the RTL designs into a single one and then verilate it.
Is that possible for them to provide their verilated models each, then instantiate the verilated SystemC models in one single SystemC simulation?

Thanks
Robert

**Comment by wsnyder** [comment]
Yes, build the models and connect in  SystemC.  The Verilator --lib-create  is an all-Verilog option too though likely a rougher process, as requires Verilator for integration and is non-standard.  I'd suggest SystemC first if that works.

**Comment by robertbu** [comment]
Thanks for the quick response. If I instantiate two verilated models for two different RTL module in one SystemC simulation, that works. However, If I instantiate the same verilated model twice in one SystemC simulation, the AXI bus interface hangs. I'll dig further.
